### DISCUSSION TITLE
Loading Template File

### DISCUSSION THREAD

**Discussion by jeffthedog** [Security-Onion-Solutions/securityonion#13339] [2.4] [answered]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

10347MB

### Storage for /

-

### Storage for /nsm

-

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hey there guys, I have been trying to download Security Onion for a few days now. I attempted many versions of ISOs and have found the most succcess so far with this current version - securityonion-2.4.80-20240624

(I followed the instructions exactly in this video https://www.youtube.com/watch?v=Uvi8WGFM2d0 )

I have been waiting for awhile now and it has been stuck on this for quite some time. Is there an error or should i wait for a bit longer? 

<img width="407" alt="Screenshot 2024-07-15 175438" src="https://github.com/user-attachments/assets/8393b296-6b0f-40d9-8836-e70ba8cbdc37">

Thanks

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment, marked as answer]
How long has it been sitting there?

Is this a physical machine or virtual machine?

How are you installing the ISO image (USB, DRAC, ILO, or other)?

What kind of storage are you using (NVME, SSD, or rotational)?

**Reply by jeffthedog** [reply]
Hey there, thanks for the quick reply! it has been loaded successfully! However, there are some missing services, hopefully you are able to help me with these issues

https://github.com/Security-Onion-Solutions/securityonion/discussions/13343

Thanks :D
